### DISCUSSION TITLE
Multiple Tabs with multiple Users

### DISCUSSION THREAD

**Discussion by MuhammedKilinc60** [pocketbase/pocketbase#2146] [Q&A]
Hello I have a  question : 
Since pocketbase is storing the auth data in localstorage it is shared between tabs and windows of the same browser.
My question is how we can login with multiple user on different tabs or windows. 

Ty

**Comment by ganigeorgiev** [comment]
The `localStorage` is shared per `protocol://host:port`.

I'm not sure that I understand your use case, but if you don't want to have a persistence storage, you can specify a `localStorage` key every time when you initialize the SDK:

```js
import PocketBase, { LocalAuthStore } from "pocketbase"

const pb = new PocketBase("http://127.0.0.1:8090", new LocalAuthStore("some_random_key"));
```

Or altenatively, create a non-persistent storage, something like:

```js
import PocketBase, { BaseAuthStore } from "pocketbase"

class NonPersistentStore extends BaseAuthStore {}

const pb = new PocketBase("http://127.0.0.1:8090", new NonPersistentStore());
```

**Reply by MuhammedKilinc60** [reply]
I mean using a website with 2 different users on 2 different pages

**Reply by ganigeorgiev** [reply]
I still don't understand what do you mean. Are the 2 different users share the same browser?

What happens if one of them close the tab window. How do you proceed with the last session only for this user when they open a new tab?

In any case, the above suggested solutions provide both non-persistent storage and option to specify a different localStorage key per tab (although this doesn't make much sense to me; I guess it could be useful for example in case you have multiple PocketBase instances serving on different subpaths).
